ideasbox/serveradmin/views.py

```diff
@@ -11,6 +14,7 @@
 from .backup import Backup
 from .utils import call_service
 
+interface = 'wlan0'
```

Comment:
Needs to be in the config file (/etc/default/ideasbox, then an env var, because it's the system to decide this).